### PR TITLE
Fuel type distribution

### PR DESCRIPTION
**Pull request by dgboss**
Adds a basic fuel type description component that contains:
 - fuel type code with tool tip containing description of the fuel type
 - a horizontal bar indicating the percent contribution of the fuel type to the overall high HFI (>4,000) area
 - tweaked some subtitles in the side panel

This PR does not include the critical hours or the ability to select fuel types. 
# Test Links:
[Landing Page](https://wps-pr-3792-e1e498-dev.apps.silver.devops.gov.bc.ca/)
[MoreCast](https://wps-pr-3792-e1e498-dev.apps.silver.devops.gov.bc.ca/morecast)
[Percentile Calculator](https://wps-pr-3792-e1e498-dev.apps.silver.devops.gov.bc.ca/percentile-calculator)
[C-Haines](https://wps-pr-3792-e1e498-dev.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-3792-e1e498-dev.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-3792-e1e498-dev.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN,s=9999&f=c7&c=NaN&w=NaN)
[Auto Spatial Advisory (ASA)](https://wps-pr-3792-e1e498-dev.apps.silver.devops.gov.bc.ca/auto-spatial-advisory)
[HFI Calculator](https://wps-pr-3792-e1e498-dev.apps.silver.devops.gov.bc.ca/hfi-calculator)

### CODE REVIEW

File: web/src/features/fba/components/viz/ElevationInfoViz.tsx [hunk 1]
```diff
@@ -19,9 +19,10 @@ const classes = {
 
 const Root = styled('div')({
   [`& .${classes.header}`]: {
```
Comment (conbrad) [inline web/src/features/fba/components/viz/ElevationInfoViz.tsx:21]: Do we need this styled div anymore?

File: web/src/features/fba/components/viz/FuelSummary.tsx [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+import { groupBy, isUndefined } from 'lodash'
+import { DateTime } from 'luxon'
+import FuelDistribution from 'features/fba/components/viz/FuelDistribution'
+import { DataGridPro, GridColDef, GridRenderCellParams } from '@mui/x-data-grid-pro'
+import { useTheme } from '@mui/material/styles'
+
+export interface FuelTypeInfoSummary {
+  area: number
+  criticalHoursStart?: DateTime
+  criticalHoursStop?: DateTime
```
Comment (conbrad) [inline web/src/features/fba/components/viz/FuelSummary.tsx:13]: Minor nit: we use "start" and "end" elsewhere

```suggestion
  criticalHoursEnd?: DateTime
```

File: web/src/features/fba/components/viz/FuelSummary.tsx [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+    flex: 3,
+    headerName: 'Area Distribution',
+    minWidth: 200,
+    sortable: false,
+    renderCell: (params: GridRenderCellParams) => {
+      console.log(params)
+      return <FuelDistribution code={params.row['code']} percent={params.row['percent']} />
+    }
+  }
+  // { field: 'hours', flex: 2, headerName: 'Critical Hours', minWidth: 150, sortable: false }
```
Comment (conbrad) [inline web/src/features/fba/components/viz/FuelSummary.tsx:49]:
```suggestion

```

**Review by conbrad — APPROVED** [review]

File: web/src/features/fba/components/viz/FuelSummary.tsx (hunk 2 above)
Comment (dgboss) [reply web/src/features/fba/components/viz/FuelSummary.tsx]: Fixed in https://github.com/bcgov/wps/pull/3792/commits/2025fad89a3a04d99b7c15d2956bb9b31b947584

File: web/src/features/fba/components/viz/FuelSummary.tsx (hunk 3 above)
Comment (dgboss) [reply web/src/features/fba/components/viz/FuelSummary.tsx]: Fixed in https://github.com/bcgov/wps/pull/3792/commits/2025fad89a3a04d99b7c15d2956bb9b31b947584

File: web/src/features/fba/components/viz/ElevationInfoViz.tsx (hunk 1 above)
Comment (dgboss) [reply web/src/features/fba/components/viz/ElevationInfoViz.tsx]: Good catch. We probably haven't needed this for quite a while. We don't need to render anything (including an empty div) when there is no elevation data. 
Fixed in https://github.com/bcgov/wps/pull/3792/commits/c06a9100674558a3c0586a75ed23d6f2f652cb33

File: web/src/features/fba/components/viz/FuelSummary.tsx [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+  },
+  {
+    field: 'area',
+    flex: 3,
+    headerClassName: 'fuel-summary-header',
+    headerName: 'Distribution > 4k kW/m',
+    minWidth: 200,
+    sortable: false,
+    renderCell: (params: GridRenderCellParams) => {
+      console.log(params)
```
Comment (brettedw) [inline web/src/features/fba/components/viz/FuelSummary.tsx:48]: Left over console.log?

**Review by brettedw — APPROVED** [review]
Nice work, looks fantastic

File: web/src/features/fba/components/viz/FuelSummary.tsx (hunk 4 above)
Comment (dgboss) [inline web/src/features/fba/components/viz/FuelSummary.tsx:48]:
```suggestion
```

**Pull request merged**